CHANGELOG.md

```diff
@@ -95,6 +95,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix hiding the pagination of data tables
 - Fix shared objects functionality: visibility, deleting, editing tags
 
+### Removed
+
+- Unused components and views
```

Comment:
would be good to name the views, as it is easier to track

Reply:
updated